pages/governance.md

```diff
@@ -0,0 +1,110 @@
+* Represent JuMP in interactions with NumFOCUS, e.g., by attending NumFOCUS's
+  annual summit.
+
+The current members of the steering committee are:
+
+* [Miles Lubin (Google)](https://github.com/mlubin)
+* [Juan Pablo Vielma (Google)](https://github.com/juan-pablo-vielma)
+* [Joey Huchette (Rice University)](https://github.com/joehuchette)
+* [Oscar Dowson](https://github.com/odow)
+* [Changhyun Kwon (U. South Florida)](https://github.com/chkwon)
+
+If a member of the steering committee leaves the committee, the remaining
+members will invite a member of the community to join in order to maintain a
+quorum of five members.
+
+### Conflict of interest
+
+It is expected that Steering Council members will be employed at a wide range of
+companies, universities and non-profit organizations. Because of this, it is
+possible that Members will have conflicts of interest. Such conflicts of
+interest include, but are not limited to:
+
+* Financial interests, such as investments, employment or contracting work,
+  outside of the Project that may influence their work on the Project.
+* Access to proprietary information of their employer that could potentially
+  leak into their work with the Project.
+
+All members of the Council shall disclose any conflict of interest they may
+have. Members with a conflict of interest in a particular issue may participate
+in Council discussions on that issue, but must recuse themselves from voting on
+the issue.
+
+## Channels of community involvement
+
+There are four main channels that the JuMP project uses to engage with the
+community.
+
+### Discourse forum
+
+The [Discouse forum](https://discourse.julialang.org/c/domain/opt/13) is the
+best place to go for help if you are a user of JuMP.
+
+Before posting a question, we ask that you:
+
+* Search the forum for similar questions that may have been already answered.
+* Read the first post of [PSA: make it easier to help you](https://discourse.julialang.org/t/psa-make-it-easier-to-help-you/14757/81).
+
+Remember to post a simplified version of your problem that demonstrates the
+error you are experiencing or the code you currently have, to format your code
+properly, and to provide as much detail as possible.
+
+### Gitter chatroom
+
+The [Gitter channel](https://gitter.im/JuliaOpt/JuMP-dev) is for
+developer-focused chat about JuMP. If you are hacking on JuMP and have a
+question about the internals, this is the best place to ask.
+
+If you are a user needing general help, we ask that you use the Discourse forum
+instead.
+
+### Monthly developer call
+
+The project holds a monthly developer call on the fourth Thursday of every month
+at 14:00 Eastern to discuss JuMP related business. For information on how to
+take part, join the  [Gitter channel](https://gitter.im/JuliaOpt/JuMP-dev) and
+ask `@odow` or `@mlubin` for an invite to the monthly developer call.
+
+### JuMP-dev workshops
+
+Approximately once per year, the JuMP community meets for a multi-day workshop,
```

Comment:
For a governance doc, this should talk about how the workshops are organized, e.g., the council chooses a chair.